A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_ProgressPage.js

```diff
@@ -89,7 +104,7 @@ class CDUProgressPage {
             ["\xa0POSITION UPDATE AT"],
             ["{small}*{end}[\xa0\xa0\xa0\xa0][color]cyan"],
             ["\xa0\xa0BRG / DIST"],
-            ["{small}\xa0---°/----.-{end}", "{small}TO{end} {cyan}[{sp}{sp}{sp}{sp}{sp}]{end}"],
+            [progBearingDist, progWaypoint, "{small}\xa0TO{end}"],
```

Comment:
When using the center position, the text alignment is totally off.

```suggestion
            [progBearingDist + "{small}\xa0TO{end}", progWaypoint],
```

Reply:
So this was based on the Honeywell manual and some guesswork... now I've found a couple of [real](https://youtu.be/efQlfF87c8c?t=225) [references](https://youtu.be/twwUjHXdNVU?t=760) and I don't think either my version or yours is correct. I'm going to mark this a draft for now and make this match IRL, and also look into feasibility of supporting runways as requested above.